ginga/canvas/types/basic.py

```diff
@@ -158,7 +159,7 @@ def contains_arr(self, x_arr, y_arr):
         # See: http://alienryderflex.com/polygon/
         xa, ya = x_arr, y_arr
 
-        result = numpy.empty(ya.shape, dtype=numpy.bool)
+        result = numpy.empty((ya.size, xa.size), dtype=numpy.bool)
```

Comment:
@pllim, why do `xa` and `ya` have different shapes?  `xa` (xarr) should be an array of the X values of all the points, and `ya` (yarr) should be an array of all the Y values of the points.  If there are N points, then there should be an equal number of X and Y values.

Usually when you have an array of points X, Y you can get these two arrays at "no cost" by simply taking the transpose view of the array and indexing the 0 (X) and 1 (Y) elements. i.e.:
```python
contains = obj.contains_arr(points.T[0], points.T[1])
```